torch/csrc/api/include/torch/nn/init.h

```diff
@@ -7,6 +7,28 @@ namespace torch {
 namespace nn {
 namespace init {
 
+enum Nonlinearity {
```

Comment:
Please use `enum class` instead of `enum`, see https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum